site/blog/cql-starter/cql-quickstart-cql.md

```diff
@@ -81,19 +81,22 @@ docker container logs <your-container-id>
 
```

Comment:
Minor comment, adding the comment here because github won't let me comment on unchanged lines. We might want to consider switching <your-container-id> with cass4 for consistency

Reply:
Can you reference the line in the changed file so I better understand the context of where `cass4` should be used?  Sorry, not following your suggestion.

Reply:
it's around line 81, where it says "`docker container logs <your-container-id>`". the actual container id is `cass4`. It's not that important for someone who knows how to use docker. just a suggestion.

Reply:
./nb5 --list-workloads | grep http-rest-starter

Reply:
Gotcha, understand that suggestion now.  Thanks!